src/components/navbar/HomeEmptyNav.js

```diff
@@ -1,11 +1,30 @@
 import React, { PropTypes, Component } from 'react';
 import { connect } from 'react-redux';
 import { changeStep } from '../../actions/navbarActions';
+import { getTopics } from '../../actions/topicActions';
 import smilies from '../../assets/smilies.svg';
 import UserNavBar from './UserNavBar';
 
 class HomeEmptyNav extends Component {
 
+  componentWillMount() {
+    this.props.getTopics();
+  }
+
+  renderTopics(topics) {
```

Comment:
It's a good idea, yes. Not explicity necessary, but some linters will whine about this. I'd take it out, yes.